### DISCUSSION TITLE
Where do I start to help contributing ?

### DISCUSSION THREAD

**Discussion by SkaarFacee** [immich-app/immich#12464] [General]
I want to help contributing to this amazing repository. Do you have any suggestions where I can get started? I would ideally like to contribute something related to AI or deep learning

**Comment by bo0tzz** [comment]
You can find an overview of the architecture, development setup, etc. at https://immich.app/docs/developer/architecture. For a starting point I would recommend having a look through the bugs and feature requests for something you feel like working on. If that's a larger feature, please check in with us before starting work to make sure we're all on the same page. 

If you have any questions don't hesitate to ask, either here on github or in the #contributing channel on https://discord.immich.app/
